src/bump.ts

```diff
@@ -774,3 +830,32 @@ export async function bumpSdkVer(
   core.endGroup();
   return bumped;
 }
+
+/**
+ * For SdkVer, the latest tag (i.e. "current version") may not be the starting
+ * point we want for generating a changelog; in this context, we want to get a
+ * list of commits since the last _full_ release.
+ *
+ * Returns an object containing:
+ *   - the name of the last full release reachable from our current version
+ *   - the list of valid Conventional Commit objects since that release
+ */
+async function collectChangelogCommits(
+  previousRelease: string,
+  config: Configuration
+): Promise<ConventionalCommitMessage[]> {
+  core.startGroup(`📜 Gathering changelog information`);
+  const commits = await getCommitsBetweenRefs(previousRelease);
+  core.info(
+    `Processing commit list (since ${previousRelease})` +
+      `for changelog generation:\n-> ` +
+      `${commits.map(c => c.message.split("\n")[0]).join("\n-> ")}`
+  );
+
+  const processedCommits = processCommitsForBump(commits, config);
+
+  core.endGroup();
+  return processedCommits
+    .map(c => c.message)
+    .filter(c => c) as ConventionalCommitMessage[];
```

Comment:
What is the purpose of this filter?

Reply:
This filters out falsy values; `c` can `undefined` here if it can't be parsed as a Conventional Commit.